pkgs/games/keeperrl/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv, fetchFromGitHub, SDL2, SDL2_image, openal, libvorbis, zlib, curl,
+  libGL, makeWrapper, boost, makeDesktopItem }:
+
+stdenv.mkDerivation rec {
+  pname = "keeperrl";
```

Comment:
```suggestion
  pname = "keeperrl-unstable";
```

Reply:
Or 29.1?

Reply:
Wouldn't it make more sense to include unstable in the version as `version = "unstable-2019-10-21"`?